### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by hutch4321** [PowerShell/PSReadLine#1463]
Last 1 Keys:
 &

Exception:
System.ArgumentOutOfRangeException:
The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key,
Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\covid\OneDrive\Desktop\Code Training\Lynda_Com training\Ex_Files_Learning_Python\Exercise Files>  "C:/Program Files (x86)/python.exe" "c:/Users/covid/OneDrive/Desktop/Code Training/Lynda_Com training/Ex_Files_Learning_Python/Exercise Files/Ch2/conditionals_start.py"
At line:1 char:38
+ ... python.exe" "c:/Users/covid/O
neDrive/Desktop/Code 
Training/Lynda_Com  ...
+                 ~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~
Unexpected token '"c:/Users/covid/O
neDrive/Desktop/Code 
Training/Lynda_Com training/Ex_File
s_Learning_Python/Exercise 
Files/Ch2/conditionals_start.py"' 
in expression or statement.
    + CategoryInfo          : Pars 
   erError: (:) [], ParentContain  
  sErrorRecordException
    + FullyQualifiedErrorId : Unex 
   pectedToken
 
PS C:\Users\covid\OneDrive\Desktop\Code Training\Lynda_Com training\Ex_Files_Learning_Python\Exercise Files>

**Comment by iSazonov** [comment]
See #1306

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.1 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
